contracts/cairo/deploy.sh

```diff
@@ -32,13 +30,14 @@ echo -e "- ${PINK}[ETH] Market Maker: $MM_ETHEREUM_WALLET${COLOR_RESET}"
 
 echo -e "${GREEN}\n=> [SN] Deploy Escrow${COLOR_RESET}"
 ESCROW_CONTRACT_ADDRESS=$(starkli deploy \
-  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE \
+  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE --rpc $SN_RPC_URL \
```

Comment:
I think it might be best to remove --rpc param since its implementation is being handled in PR #51 . What do you think?

Reply:
check the other answer :pray: